roles/sensu/server/defaults/main.yml

```diff
@@ -4,7 +4,8 @@ sensu_api_port: 4567
 sensu_api_server: localhost
 
 sensu_redis_server: "{{ hostvars[groups.redis_hosts.0].ansible_default_ipv4.address}}"
-sensu_redis_port: 6379
+sensu_redis_port: '{{ redis_listen_port }}'
```

Comment:
I don't understand why we have to have two variables for setting single parameter.